client/settings/transactions-and-deposits/index.js

```diff
@@ -51,9 +59,18 @@ const TransactionsAndDeposits = () => {
 						'woocommerce-payments'
 					) }
 				/>
-				<div className="general-settings__account-statement-wrapper">
+				{ customerBankStatementErrorMessage && (
+					<Notice status="error" isDismissible={ false }>
+						<span
+							dangerouslySetInnerHTML={ {
+								__html: customerBankStatementErrorMessage,
+							} }
+						/>
+					</Notice>
+				) }
+				<div className="transactions-and-deposits__account-statement-wrapper">
 					<TextControl
-						className="general-settings__account-statement-input"
```

Comment:
All these styles were prefixed with `general-settings__*`, but they were not under the "general settings" section.
Might have been due to a recent refactor - now they are prefixed with the `transactions-and-deposits__*` scope.